fmt/src/header/parser.rs

```diff
@@ -120,13 +126,15 @@ pub fn parse_header(
                     } else {
                         loop {
                             line = file_content.next_line();
-                            if let Some(l) = line.as_ref()
-                                && l.starts_with(before)
-                            {
-                                in_place_header.push_str(&l.to_lowercase());
-                                if header_def.multiple_lines && header_def.is_last_header_line(l) {
-                                    found_end = true;
-                                    break;
+                            if let Some(l) = line.as_ref() {
+                                if l.starts_with(before) {
```

Comment:
This is the boolean logic error. The condition of else below is changed.

I'm patching this PR now, so you can wait for my follow-up.